public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -38,216 +26,124 @@ export const matchesWidth = (width: number) => window.matchMedia(`(max-width: ${
-    return (
-      <>
-        <Stack gap={1}>
-          {buttons.map((button, index) => (
-            <React.Fragment key={index}>{button}</React.Fragment>
-          ))}
-          {moreActions.length > 0 && (
-            <Dropdown
-              overlay={
-                <Menu>
-                  {moreActions.map((action) => (
-                    <React.Fragment key={uniqueId('action_')}>{action}</React.Fragment>
-                  ))}
-                </Menu>
-              }
-            >
-              <Button variant="secondary" size="sm">
-                More
-                <Icon name="angle-down" />
-              </Button>
-            </Dropdown>
-          )}
-        </Stack>
-        {!!ruleToDelete && (
-          <ConfirmModal
-            isOpen={true}
-            title="Delete rule"
-            body={
-              <div>
-                <p>
-                  Deleting &quot;<strong>{ruleToDelete.name}</strong>&quot; will permanently remove it from your alert
-                  rule list.
-                </p>
-                <p>Are you sure you want to delete this rule?</p>
-              </div>
-            }
-            confirmText="Yes, delete"
-            icon="exclamation-triangle"
-            onConfirm={deleteRule}
-            onDismiss={() => setRuleToDelete(undefined)}
-          />
-        )}
-
-        {redirectToClone && (
-          <RedirectToCloneRule
-            identifier={redirectToClone.identifier}
-            isProvisioned={redirectToClone.isProvisioned}
-            onDismiss={() => setRedirectToClone(undefined)}
-          />
-        )}
-      </>
+    buttons.push(
+      <LinkButton
+        tooltip={compact ? 'Edit' : undefined}
+        tooltipPlacement="top"
+        title={'Edit'}
+        className={buttonClasses}
+        size={buttonSize}
+        key="edit"
+        variant="secondary"
+        icon="pen"
+        href={editURL}
+      >
+        {!compact && 'Edit'}
+      </LinkButton>
     );
   }
 
-  return null;
+  return (
+    <Stack gap={1}>
+      {buttons}
+      <AlertRuleMenu
+        buttonSize={buttonSize}
+        rule={rule}
+        identifier={identifier}
+        showCopyLinkButton={showCopyLinkButton}
+        handleDelete={() => showDeleteModal(rule)}
+        handleDuplicateRule={() => setRedirectToClone({ identifier, isProvisioned })}
+        onPauseChange={() => {
+          // Uses INSTANCES_DISPLAY_LIMIT + 1 here as exporting LIMIT_ALERTS from RuleList has the side effect
+          // of breaking some unrelated tests in Policy.test.tsx due to mocking approach
+          const limitAlerts = hasActiveFilters ? undefined : INSTANCES_DISPLAY_LIMIT + 1;
+          // Trigger a re-fetch of the rules table
+          // TODO: Migrate rules table functionality to RTK Query, so we instead rely
+          // on tag invalidation (or optimistic cache updates) for this
+          dispatch(fetchAllPromAndRulerRulesAction(false, { limitAlerts }));
```

Comment:
Pausing is supported only by GMA, so it should be enough to fetch Prom and Ruler rules for grafana data source only

Reply:
I think I've done it now, by dispatching two other events rather than this one - can you check please?